modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TableRowSkinTest.java

```diff
@@ -82,7 +84,7 @@ public void before() {
 
         tableView.setItems(items);
 
-        stageLoader = new StageLoader(tableView);
+        stageLoader = new StageLoader(new Scene(tableView));
```

Comment:
this changes the context of unrelated tests - no idea whether or not it matters, but I would try not to :)

Reply:
Good catch, actually this changes is a leftover after some changes I did while getting the test to fail... but now I see it is not needed after all.